xla/stream_executor/cuda/cuda_blas_lt.cc

```diff
@@ -465,17 +464,44 @@ absl::Status BlasLt::MatmulPlan::DoMatmul(
 
     std::unique_ptr<ActivateContext> activation = blas_lt->parent_->Activate();
 
-    void* c_ptr = args.c.opaque();
-    if (beta_ == 0.0) {
-      c_ptr = nullptr;
-    }
-
     if (palgo != nullptr) {
+#if CUDA_VERSION >= 13000
```

Comment:
Is this version check really needed? Can't we use the same code path for older CUDA versions?

Reply:
Sure, that would probably be better. Let me fix it.